pkg/connector/connector.go

```diff
@@ -36,7 +37,7 @@ func (d *Connector) Asset(ctx context.Context, asset *v2.AssetRef) (string, io.R
 // Metadata returns metadata about the connector.
 func (d *Connector) Metadata(ctx context.Context) (*v2.ConnectorMetadata, error) {
 	return &v2.ConnectorMetadata{
-		DisplayName: "My Baton Connector",
+		DisplayName: "Baton Snowflake",
```

Comment:
😆 oops.

Maybe we want to change the description too.

Reply:
Fixed